docs/src/main/sphinx/release/release-447.md

```diff
@@ -0,0 +1,47 @@
+# Release 447 (8 May 2024)
+
+## General
+
+* Add support for {doc}`/sql/show-create-function`. ({issue}`21809`)
+* Add support for the {func}`bitwise_xor_agg` aggregation function. ({issue}`21436`)
+* {{breaking}} Require JDK 22 to run Trino. ({issue}`20980`)
+* Improve performance of `ORDER BY` queries with `LIMIT` on large data sets. ({issue}`21761`)
+* Improve performance of queries containing the {func}`rank` or
+  {func}`row_number` window functions . ({issue}`21639`)
+* Improve performance of correlated queries with `EXISTS`. ({issue}`21422`)
+
+## CLI
+
+* Fix incorrect error location markers for SQL routines causing CLI to print
+  exceptions. ({issue}`21357`)
+
+## Delta Lake connector
+
+* Add support for concurrent `DELETE` and `TRUNCATE` queries. ({issue}`18521`)
+* Fix under-accounting of memory usage when writing strings to Parquet files. ({issue}`21745`)
```

Comment:
under-accounting .. is that a word?

Reply:
It is now! I think it's the most concise way to convey the concept of underestimation of memory accounting, which is a mouthful.

Reply:
haha.. thats what I thought too .